core/api/src/domain/pagination/index.types.d.ts

```diff
@@ -0,0 +1,19 @@
+type PaginatedResult<T> = import("graphql-relay").Connection<T>
```

Comment:
Pagination is not really its own domain. Also domain code should not have an external dependency - especially not something as heavy as a IO layer framework.